llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1506,18 +1506,17 @@ static void transformRecipestoEVLRecipes(VPlan &Plan, VPValue &EVL) {
                   })
               .Case<VPWidenCastRecipe>(
                   [&](VPWidenCastRecipe *CInst) -> VPRecipeBase * {
```

Comment:
The name here is also confusing, as `Inst` implies an instructions, while this is a cast recipe... Better name it `CastR`